src/api.yaml

```diff
@@ -0,0 +1,127 @@
+constant:
+  array layer count undefined: Undefined value for array layer count
```

Comment:
Maybe explain that undefined will mean some default value is used (and refer to the WebGPU spec?)

Reply:
Ditto for other undefined values

Reply:
Ah will do, didn't realize that that is what would happen. Will look up in the WebGPU spec what they end up meaning.

Reply:
Added a note that a default would be used. Haven't added the spec references yet as I don't know where they should link, and don't have the code to link them at the moment, heh.